cli/tests/awsbatch/data/aws_api_responses/batch_describe-jobs_single_mnp_job.json

```diff
@@ -0,0 +1,65 @@
+{
+  "jobs": [
+    {
+      "status": "SUCCEEDED",
+      "parameters": {},
+      "jobDefinition": "arn:aws:batch:us-east-1:653949452088:job-definition/parallelcluster-mnp-final-mnp:2",
+      "statusReason": "Essential container in task exited",
+      "jobId": "6abf3ecd-07a8-4faa-8a65-79e7404eb50f",
+      "attempts": [
+        {
+          "startedAt": 1543836512293,
+          "container": {
+            "networkInterfaces": [],
+            "logStreamName": "parallelcluster-mnp-final-mnp/default/3d430bca-d7b4-4436-9efe-ae5f76a0b81b",
+            "exitCode": 0
+          },
+          "stoppedAt": 1543836586008,
+          "statusReason": "Essential container in task exited"
+        }
+      ],
+      "nodeProperties": {
+        "nodeRangeProperties": [
+          {
+            "targetNodes": "0:1",
+            "container": {
+              "mountPoints": [],
+              "image": "653949452088.dkr.ecr.us-east-1.amazonaws.com/paral-docke-150d1t4y4o4xj:alinux",
+              "environment": [
+                {
+                  "name": "SHARED_DIR",
+                  "value": "/shared"
+                },
+                {
+                  "name": "MASTER_IP",
+                  "value": "10.0.0.47"
+                }
```

Comment:
please add: 
```
{ 
   "name": "PCLUSTER_JOB_S3_URL",
    "value": "s3://parallelcluster-xxx/batch/job-xxx"
}
```
It is required since https://github.com/aws/aws-parallelcluster/pull/733/files has been merged.